backend/tests/index.test.js

```diff
@@ -0,0 +1,40 @@
+const { expect } = require( 'chai' );
+const { UserHelper } = require( '../src/helpers/UserHelper' );
+
+const dataFromRedis = {
```

Comment:
Rename it. This unit test is not aware of redis existance.

`sessionsData`?

Reply:
Also move the test file to: tests/helpers/UserHelper.js

Reply:
Done.